viewer/lib/utils.electron.js

```diff
@@ -2,12 +2,12 @@ const THREE = require('three')
 const path = require('path')
 
 function loadTexture (texture, cb) {
-  const url = path.resolve(__dirname, '../public/' + texture)
+  const url = path.resolve(__dirname, '../../public/' + texture)
```

Comment:
could we use something like `require.resolve('prismarine-viewer/public/')` here ?

Reply:
yeah done